code/modules/clothing/clothing.dm

```diff
@@ -85,6 +85,7 @@ BLIND     // can't see anything
 	put_on_delay = 40
 	var/mask_adjusted = 0
 	var/ignore_maskadjust = 1
+	var/hathead_only = 1

```

Comment:
This is snowflake. It only works for masks that can also be hats. I suggest a more general system. Something like `var/adjusted_flags = null` and then on the adjust_mask code:
```
if(adjusted_flags)
    slot_flags = adjusted_flags
```